packages/askar/src/utils/askarStoreConfig.ts

```diff
@@ -48,8 +48,7 @@ export const uriFromStoreConfig = (
       database.credentials.password
     )}@${database.config.host}/${encodeURIComponent(storeConfig.id)}`
   } else {
-    // @ts-expect-error
-    throw new WalletError(`Storage type not supported: ${database.type}`)
+    throw new AskarError(`Storage type not supported: ${database}`)
```

Comment:
The ts expect error is there on purpose as the `database.type` does not exist (since we check all possible values). 

Not sure if we want to print the `database` instance as the error?

Maybe just `Storage type not supported` would be enough? There's not really a case in the near future i think where we will extend the storage type